ramls/user.json

```diff
@@ -120,15 +120,15 @@
                 "type": "boolean"
               }
             },
-            "additionalProperties": false
```

Comment:
Do you think it makes sense to allow all additional properties? As I see, only one additional properties is used, it's two levels of json schema hierarchy higher